Adafruit_Microbit.h

```diff
@@ -1,102 +0,0 @@
-/*!
```

Comment:
dont remove this file

Reply:
Yeah, I'm not sure what happened there. I've readded the file with the change.